challengeII/script.js

```diff
@@ -0,0 +1,27 @@
+function tbl() {
+  if((document.getElementById("name").value != "")&&
+  (document.getElementById("food").value != "") &&
+  (document.getElementById("color").value) != ""){
+    var para = document.createElement("tr");
+    var nodea = document.createElement("td");
+    var nodeb = document.createElement("td");
+    var nodec = document.createElement("td");
+    para.appendChild(nodea);
+    para.appendChild(nodeb);
+    para.appendChild(nodec);
+    var nodex = document.createTextNode(document.getElementById("name").value);
+    var nodey = document.createTextNode(document.getElementById("food").value);
+    var nodez = document.createTextNode(document.getElementById("color").value);
+    nodea.appendChild(nodex);
+    nodeb.appendChild(nodey);
+    nodec.appendChild(nodez);
+    var element = document.getElementById("tabl");
+    element.appendChild(para);
+    document.getElementById("name").value = ""
+    document.getElementById("food").value = ""
+    document.getElementById("color").value = ""
+    document.getElementById("nope").textContent = ""
```

Comment:
Using jQuery and ES6 the syntax is a bit shorter. I've stored the values in variables to make the code a little easier to read. This code would replace lines 5-23:

```
        const enteredName = $("#name").val();
        const enteredFood = $("#food").val();
        const enteredColor = $("#color").val();

        $("table#tabl").append(`<tr>
            <td>${enteredName}</td>
            <td>${enteredFood}</td>
            <td>${enteredColor}</td>
            </tr>`);
```